### ISSUE TITLE
Debug Toolbar Button has too low contrast

### ISSUE THREAD

**Issue by jribbens** [jazzband/django-debug-toolbar#1981]
When the page background is white (obviously very common), the opacity and colouring of the debug toolbar button is such that the colour contrast of the text is too low as per accessibility rules (it's 2.61, it should be >=3). If the opacity is increased from 0.5 to 0.6 then this issue will be fixed.

(The low-contrast is an accessibility issue in and of itself, but also it means that when developers are performing accessibility testing during development, they will get warnings about the low contrast of the button, which are distracting and spurious since the button will of course not be present on the production site.)
